### PR TITLE
Bug 2053622: PodDisruptionBudgetAtLimit should not alert when no app/pods exist

### PR DESCRIPTION
**Pull request by atiratree**
meaning alert should not fire when currentHealthy == 0 and desiredHealthy == 0

It makes sense to not fire an alert if there is no application to disrupt

### CODE REVIEW

**Review by ravisantoshgudimetla — APPROVED** [review]
/lgtm

Thank you for working on this @atiratree

**Review by soltysh — APPROVED** [review]
/lgtm
/hold cancel

File: manifests/0000_90_kube-controller-manager-operator_05_alerts.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ spec:
             summary: The pod disruption budget is preventing further disruption to pods.
             description: The pod disruption budget is at minimum disruptions allowed level. The number of current healthy pods is equal to desired healthy pods.
           expr: |
-            max by(namespace, poddisruptionbudget) (kube_poddisruptionbudget_status_current_healthy == kube_poddisruptionbudget_status_desired_healthy)
+            max by(namespace, poddisruptionbudget) (kube_poddisruptionbudget_status_current_healthy == kube_poddisruptionbudget_status_desired_healthy and on (namespace, poddisruptionbudget) kube_poddisruptionbudget_status_expected_pods > 0)
```
Comment (wking) [inline manifests/0000_90_kube-controller-manager-operator_05_alerts.yaml:29]: nit: can you accomplish this same goal more simply with:

```
max by(namespace, poddisruptionbudget) (kube_poddisruptionbudget_status_current_healthy == kube_poddisruptionbudget_status_desired_healthy > 0)
```

**Pull request merged**